travis_docker_test_script.sh

```diff
@@ -10,10 +10,4 @@ PKGNAME="PGFPlotsX"
 cd /mnt && if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
 
 # run tests
-$JULIABIN -e "import Pkg; Pkg.clone(\"/mnt/\", \"$PKGNAME\"); Pkg.build(\"$PKGNAME\"); Pkg.test(\"$PKGNAME\"; coverage=true)"
-TEST_EXIT=$?                    # return with this
-
-# save coverage results back to host
-PKGDIR=`$JULIABIN -e "import Pkg, $PKGNAME; print(joinpath(dirname(pathof($PKGNAME)), ".."))"`
-rsync -mav --include="*/" --include="*.cov" --exclude="*" $PKGDIR/ /mnt/
```

Comment:
I believe this is not needed since `$PKGDIR` and `/mnt` are the same directory.

Reply:
I think you are right.